client/src/services/psi-cohort.js

```diff
@@ -101,6 +101,22 @@ export const fetchCohorts = async ({ psiId }) => {
   throw new Error(response.error || response.statusText || 'Unable to load cohorts details');
 };
 
+export const fetchCohort = async ({ cohortId }) => {
+  const response = await fetch(`${API_URL}/api/v1/cohorts/${cohortId}/participants`, {
```

Comment:
This route structure would make me expect a list of participants, not cohort details, why not just 

`/cohorts/${cohortId}`